### DISCUSSION TITLE
Application for Joining The Team

### DISCUSSION THREAD

**Discussion by OmegaBeam69** [ROM-builders/temporary#13700] [Q&A]
GitHub: https://GitHub.com/WasikFahim69
My successful build Xdroid: https://cirrus-ci.com/task/6556970290249728
DT: https://github.com/WasikFahim69/device_xiaomi_raphael/tree/12l
KT:  https://github.com/WasikFahim69/kernel_xiaomi_raphael
VT: https://github.com/WasikFahim69/vendor_xiaomi_raphael
TG: @OmegaBeam

**Comment by Apon77** [comment]
seems fake accounts

**Reply by OmegaBeam69** [reply]
I can dm you my details. If you want. And I'm not fake. We already talked before about cirrus.

**Reply by Apon77** [reply]
Please check #team and #learn instruction in telegram.
Also please match telegram and github name or username

**Reply by OmegaBeam69** [reply]
TG username changed as per github brother. Please check thanks
